pkg/kubernetes/components.go

```diff
@@ -37,17 +37,14 @@ func PrintComponents(name, outputFormat string) error {
 		}
 
 		list, err := client.ComponentsV1alpha1().Components(meta_v1.NamespaceAll).List(meta_v1.ListOptions{})
-		if err != nil {
-			// This means that the Dapr Components CRD is not installed and there
-			// is are therefore no component items.
-			if serr, ok := err.(*apierrors.StatusError); ok &&
-				serr.ErrStatus.Reason == meta_v1.StatusReasonNotFound {
-				list = &v1alpha1.ComponentList{
```

Comment:
LGTM, there's no change needed but in general `Items` is a slice and as such doesn't need to be initialized with `[]v1alpha1.Component`